### DISCUSSION TITLE
Fehler beim lokalem Test des Frontends - Error during local test of the frontend

### DISCUSSION THREAD

**Discussion by RLeo** [Bootstrap-Academy/Bootstrap-Academy#84] [Q&A] [answered]
Hallo zusammen,

ich habe das Frontend ausgecheckt und lokal gestartet. Beim besuch des Profils allerdings bekomme ich ein Fehler:
`SyntaxError: ambiguous indirect export: GetUnseenLectureResponse [vue-router.mjs:3471:20](http://localhost:3000/_nuxt/node_modules/vue-router/dist/vue-router.mjs)` und ` error caught during app initialization Error: ambiguous indirect export: GetUnseenLectureResponse
Caused by: SyntaxError: ambiguous indirect export: GetUnseenLectureResponse`

eigentlich habe ich noch gar nicht angefangen irgendwas zu ändern: 
![grafik](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/11270911/20db4b3c-abff-4819-8c74-8b9ec83c044d)

Was könnte ich falsch gemacht haben?

Hello everyone,

I have checked out the frontend and started it locally. However, when visiting the profile I get an error:
`SyntaxError: ambiguous indirect export: GetUnseenLectureResponse [vue-router.mjs:3471:20](http://localhost:3000/_nuxt/node_modules/vue-router/dist/vue-router.mjs)` and ` error caught during app initialization Error: ambiguous indirect export: GetUnseenLectureResponse
Caused by: SyntaxError: ambiguous indirect export: GetUnseenLectureResponse`

I haven't actually started to change anything yet: 
![grafik](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/11270911/20db4b3c-abff-4819-8c74-8b9ec83c044d)

What could I have done wrong?

**Comment by Defelo** [comment, marked as answer]
Hi, der Fehler hast sich anscheinend letzte Woche durch nen Pull Request eingeschlichen, danke fürs melden! Sollte jetzt hoffentlich gefixt sein, also probiers am besten nochmal aus und sag Bescheid, obs geklappt hat

**Reply by RLeo** [reply]
Vielen Dank das hat geholfen
